modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,314 @@
+= Installing on Kubernetes
+
+This guide demonstrates how to deploy a Sync Gateway cluster on Kubernetes.
+
+== Prerequisites
+
+Before you begin, you must have the following:
+
+* A Couchbase Server cluster already running on Kubernetes.
```

Comment:
@jamiltz  Not sure why I don't see the link to the guide when I view file....

Reply:
@jamiltz OK. But the other links seemed to work - curious why we can't use the same approach? Don't have a strong opinion but wondering...